xchange-stream-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingMarketDataService.java

```diff
@@ -1,5 +1,6 @@
 package info.bitrich.xchangestream.binance;
 
+import static info.bitrich.xchangestream.binance.dto.BaseBinanceWebSocketTransaction.BinanceWebSocketTypes.DEPTH_UPDATE;
```

Comment:
is this needed?

Reply:
Nope, your true. Together with import of array list.

Reply:
Can you please clarify one moment? sometime ago, the library did auto-formatting on the tests step, is it the same now?

Reply:
not following, this is just to remove these unused classes